docs/advanced/lido_csm_walkthrough.md

```diff
@@ -0,0 +1,222 @@
+---
+sidebar_position: 5
+description: Create a DV within Lido CSM 
+---
+# Walkthough: Deploy a CSM DV with Linux CLI
+
+To start, this guide makes a couple assumptions:
+
+1.  You're running Linux and you've installed Git and Docker. 
+2.  You'll be delpoying on mainnet. Some screenshots in this guide are from Holesky, so please verify mainnet addresses.
+
+## Getting started
+
+This guide will be broken down into 3 parts:
+
+Part 1: Create cluster multi-sig + 0xSplits contract
+
+Part 2: Use Obol Launchpad + CLI to create the cluster
+
+Part 3: Deploy the validator to CSM 
+
+
+In this guide we'll be using CSM with `extendedManagerPermissions` where the `managerAddress` is set to the cluster multi-sig and the `rewardAddress` is set to the 0xSplits contract. 
+
+
+## Part 1: Creating the Cluster multi-sig + 0xSplits Contract
+
+Detailed instructions on how to create a Safe Wallet can be found [here](https://help.safe.global/en/articles/40868-creating-a-safe-on-a-web-browser). 
+
+Squad leader should obtain the signer addresses from all the cluster members, then connect their signer wallet and create a new Safe. 
+
+![chrome_ofxRcHQItb](https://hackmd.io/_uploads/HJImiiVh0.png)
```

Comment:
Why are we not hosting the images in our github? Similarly to how it's done for the `CSM_launchpad_tab.png` image?

Reply:
We could, but it's a lot of images... wasn't sure if we have a policy here to try and save space in the docs folder or something

Reply:
I doubt it... I'd much rather our docs don't break after a year because some other service is down.